CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+
+### Responsibilities
+
+- Ensure cross-platform compatibility for every change that's accepted. Windows, Mac, Debian & Ubuntu Linux.
+- Ensure that any new code you write is covered by tests.
+- Create issues for any changes and enhancements. Discuss things transparently and get community feedback.
+- Keep changes as small as possible to ease the burden of code review.
+- Be welcoming to newcomers and encourage diverse new contributors from all backgrounds. See the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
+
+## Your First Contribution
+
+Never contributed to open-source before? Unsure of where to begin contributing to `django-easy-audit`? You can start by looking through these resources:
+
+- [Make a Pull Request](http://makeapullrequest.com/)
+- [First Timers Only](http://www.firsttimersonly.com/)
+- [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+
+If you're still unsure, feel free to ask for help! Everyone starts as a beginner.
+
+## Making changes
+
+For something that is bigger than a one or two line fix:
+
+1. Create your own fork of the code.
+2. Make the changes in your fork.
+3. Create a pull request from your fork against the main branch.
+
+Small contributions such as fixing spelling errors, where the content is small enough to not be considered intellectual property, can be submitted as a patch without forking.
+
+As a rule of thumb, changes are obvious fixes if they do not introduce any new functionality or creative thinking. As long as the change does not affect functionality, some likely examples include the following:
+
+- Spelling / grammar fixes
+- Typo correction, white space and formatting changes
+- Comment clean up
+- Bug fixes that change default return values or error codes stored in constants
+- Adding logging messages or debugging output
+- Changes to ‘metadata’ files like .gitignore, build scripts, etc.
+- Moving source files from one directory or package to another
+
+## Reporting bugs
+
+### Security vulnerabilities
+
+If you find a security vulnerability, **DO NOT** open an issue. Email [natancalzolari@gmail.com](mailto:natancalzolari@gmail.com) instead so as to not expose the vulnerability to the public.
+
+In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+
+- Can I access something that's not mine, or something I shouldn't have access to?
+- Can I disable something for other people?
+
+If the answer to either of those two questions are _yes_, then you're probably dealing with a security issue. Note that even if you answer _no_ to both questions, you may still be dealing with a security issue, so if you're unsure just send an email.
+
+### Bugs
+
+When filing an issue, make sure to answer these five questions:
```

Comment:
We could add an issue template for this in a future PR

Reply:
Agreed, that would be a good call.